src/datablock/models.jl

```diff
@@ -59,3 +59,38 @@ function blockmodel(inblock::ImageTensor{N}, outblock::KeypointTensor{N}, backbo
     head = Models.visionhead(outch, prod(outblock.sz)*N, p = 0.)
     return Chain(backbone, head)
 end
+
+"""
+    blockmodel(inblock::TableRow{M, N}, outblock::Union{Continuous, OneHotTensor{0}}, backbone=nothing) where {M, N}
```

Comment:
Can we make it clear that `backbone` is an optional argument and what the default is?

Reply:
Sure I'll do that for `blockmodel`. I was wondering if there is way for `backbone` to still be optional if someone is using `methodmodel` or maybe even `methodlearner`.

Reply:
Could even have a helper `default_tabular_backbones()` that makes the default `NamedTuple` in [this suggestion](https://github.com/FluxML/FastAI.jl/pull/141/files#r689111096)